clang/test/CodeGen/linux-kernel-struct-union-initializer.c

```diff
@@ -0,0 +1,219 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-unknown -std=gnu11 -verify -emit-llvm %s -o - | FileCheck %s
```

Comment:
update_cc_test_checks.py, maybe?  Unless it doesn't generate clear checks for some reason.

Reply:
I didn't know that. Now use the script to generate checks.